### PR TITLE
Fix time_elapsed being 0

### PR DESCRIPTION
**Pull request by yinweisu**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tabular/src/autogluon/tabular/models/rf/rf_model.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def _fit(self,
                     model = model_cls(**params)
             model = model.fit(X, y, sample_weight=sample_weight)
             if (i == 0) and (len(n_estimator_increments) > 1):
-                time_elapsed = time.time() - time_train_start
+                time_elapsed = min(time.time() - time_train_start, 0.001)  # avoid it being too small and being truncated to 0
```
Comment (liangfu) [inline tabular/src/autogluon/tabular/models/rf/rf_model.py:202]: min(0, 0.001) would still go to 0, consider max() instead ?

Comment (liangfu) [reply tabular/src/autogluon/tabular/models/rf/rf_model.py:202]: or, a better solution could be `xxx/time_elapsed` -> `xxx/(time_elapsed+1e-6)`

Comment (yinweisu) [reply tabular/src/autogluon/tabular/models/rf/rf_model.py:202]: oh... what I was doing...

**Review by liangfu — APPROVED** [review]
LGTM

**Pull request merged**